src/test/daml/Daml/Finance/Settlement/Test/Intermediated.daml

```diff
@@ -189,75 +192,65 @@ run useDelegatee = script do
           exerciseCmd (coerceInterfaceContractId @Disclosure.I buyerInstructionCid)
             Disclosure.AddObservers with
               disclosers = singleton buyer; observersToAdd = observerContext
-        Account.submitExerciseInterfaceByKeyCmd @Disclosure.I [buyer] [] buyerCashAccount
-          Disclosure.AddObservers with
-            disclosers = singleton buyer; observersToAdd = observerContext
+        buyerAccountCid <- fst <$> queryAnyByKey @Account.T @Account.I [buyer] buyerCashAccount
+        submitMulti [buyer] [] do
```

Comment:
```suggestion
        submit buyer do
```

Reply:
also in other parts of the code

Reply:
will look for regex `submitMulti \[\w+\] \[\]` and replace accordingly in the "gardening" PR